packages/cactus-plugin-ledger-connector-fabric/package.json

```diff
@@ -58,6 +58,7 @@
     "@hyperledger/cactus-core": "1.0.0",
     "@hyperledger/cactus-core-api": "1.0.0",
     "axios": "0.21.4",
+    "amqp-ts": "1.8.0",
```

Comment:
@RafaelAPB Same comment here as I made for the besu connector (I won't spam you twice with all the same comments, but the requests are the same)